src/disco/utils.js

```diff
@@ -0,0 +1,7 @@
+import { sanitizeHTML } from 'core/utils';
+import purify from 'disco/purify';
+
+
+export const sanitizeHTMLWithNewTabLinks = (text, allowTags = []) => {
+  return sanitizeHTML(text, allowTags, purify);
```

Comment:
It would help to add a comment saying that `purify` is configured to fix link targets since it's not totally obvious unless you think to dive into `disco/purify`.